test/support/fake_consumer.ex

```diff
@@ -18,6 +18,7 @@ defmodule FakeConsumer do
       :reject -> :reject
       :reply -> {:reply, :data}
       :exception -> raise "Exception happen"
+      _other -> :ok
```

Comment:
What this could be, can you please add inline comment?

Reply:
It's just a passthrough for any message. `FakeConsumer` is used only for testing, and we generally do not care about the payload, we can test that with another consumer that expects a specific payload and sends a response accordingly.